front/src/Api/iframe/ui.ts

```diff
@@ -42,17 +42,21 @@ export interface ActionMessageOptions {
     callback: () => void;
 }
 
-export interface RemotePlayerInterface {
-    addAction(key: string, callback: Function): void;
-}
-
-export class RemotePlayer implements RemotePlayerInterface {
-    private id: number;
+export class RemotePlayer {
+    public readonly id: number;
```

Comment:
I only realize now that "id" should remain private (or not passed at all).
It is a short integer representing the player in the current game, but we should not leak it to thrid parties (including the scripting API).

The UUID is what represents the user to the outside world and is the only thing we should care about outside of WorkAdventure internals.

The problem is we are already using it :(

We should probably mark it as deprecated.